paper/paper.md

```diff
@@ -9,12 +9,16 @@ tags:
 authors:
  - name: Christos Theodoropoulos
    orcid: 0000-0002-5395-0347
-   affiliation 1: Hellenic Centre for Marine Research
-   affiliation 2: National Technical University of Athens
+   affiliation: 1, 2
  - name: Nikolaos Skoulikidis
```

Comment:
We strongly prefer that authors add ORCIDs if they have an academic affiliation.